src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/Api/Methodologies/UpdateMethodologyRequest.cs

```diff
@@ -12,9 +15,14 @@ public class UpdateMethodologyRequest
 
         [Required] public string Title { get; set; }
 
-        public DateTime? PublishScheduled { get; set; }
+        [RegularExpression(@"(^\d{4}-\d{2}-\d{2}$)", ErrorMessage = "Invalid date format. Expected yyyy-MM-dd")]
```

Comment:
Is there no equivalent of Spring's `DateTimeFormat` we could be using here? If not could we have a custom validator?

Reply:
I don't think there is so I wrote a custom validator.

I think the problems I've had with this field is because I want the requested field to be a `string` to prove a point that it's not a `DateTime` while providing a convenience computed method to get it as the "start of day" UTC time for use in the service.